torch/_inductor/remote_cache.py

```diff
@@ -244,8 +244,7 @@ class RedisRemoteCacheBackend(RemoteCacheBackend[bytes]):
     def __init__(self, cache_id: str) -> None:
         super().__init__()
         if not redis:
-            # We had trouble importing redis - just skip init.
-            return
+            raise Exception("redis not available but required for remote cache")
```

Comment:
```suggestion
            raise ImportException("redis not available but required for remote cache")
```
Or at least RuntimeException. Please do not raise bare Exception